package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Some beautiful app",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Why this change?

Reply:
This line was added by default and now I know that I won use it.